tests/regression/15-deadlock/03-triple_deadlock.c

```diff
@@ -1,4 +1,4 @@
-// SKIP PARAM: --set ana.activated[+] deadlock
```

Comment:
Curiously, this was skipped before, because the transitive shortcutting meant that the pairwise forbidden order printing couldn't print all three.